### ISSUE TITLE
add descriptions

### ISSUE THREAD

**Issue by fredszaq** [snipsco/queries-ontology#4]
I would be nice to have description fields in domain, intent and slot objects. This would enable us to include meaningful documentation in the generated classes (as they will be part of the public API)

**Comment by JDureau** [comment]
With pleasure. We'll work on that tomorrow.

**Comment by alaa-saade** [comment]
Yes this is a good idea. The descriptions should be easy to infer from the examples.
